### DISCUSSION TITLE
Unable to get education account

### DISCUSSION THREAD

**Discussion by natejd6** [community/community#130657] [GitHub Education]
### Hi everyone,

Hi everyone,I am 13 and I am trying to set up an education account but my school does not support it. So can I still get it or am I just non-applicable.

**Comment by davevad93** [comment]
Hi @NathanDent6 ,

If you read the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), it says that to be eligible for GitHub Education, including GitHub Student Developer Pack and other benefits, you must:

- Be currently enrolled in a degree or diploma granting course of study such as a high school, secondary school, college, university, homeschool, or similar educational institution.
- Have a verifiable school-issued email address or upload documents that prove your current student status.
- Have a [GitHub personal account](https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github).
- Be at least 13 years old.

So basically, if you don't meet any of those requirements, you won't be able to access GitHub Education benefits.
